PWGCF/Flow/Tasks/flowPbPbpikp.cxx

```diff
@@ -8,6 +8,7 @@
 // In applying this license CERN does not waive the privileges and immunities
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
+
 /// \file FlowPbPbpikp.cxx
```

Comment:
As the filename was changed use the new one in here

Reply:
Also, the `lint` warning referring to the `name/workflow-file` is due to that it is expected that the name of the main task within the file should match the name of the file so you might use 
```
struct FlowPbPbpikp
```
for your task in the new L.61